docs/pages/api-reference/sink_connectors/s3.md

```diff
@@ -0,0 +1,89 @@
+---
+title: S3
+order: 0
+status: published
+---
+### S3
+Data connector to sink data to S3.
+
+#### Account Parameters
+<Expandable title="name" type="str">
+A name to identify the sink. The name should be unique across all Fennel connectors.
+</Expandable>
+
+<Expandable title="aws_access_key_id" type="Optional[str]" defaultVal="None">
```

Comment:
Can this take secret? Could you do the same check for all other fields added for sinks?

Reply:
@hemanthk269 would be updating the documentation with Secrets entirely soon